mlrun/runtimes/nuclio/api_gateway.py

```diff
@@ -284,7 +284,21 @@ def _validate_functions(
         function_names = []
         for func in functions:
             if isinstance(func, str):
-                function_names.append(func)
+                # check whether the function was passed as a URI or just a name
+                parsed_project, function_name, tag, hash_key = (
+                    mlrun.common.helpers.parse_versioned_object_uri(func)
+                )
+
+                if parsed_project and function_name:
```

Comment:
if function_name is not None that it wasn't a URI?
If so - explain this in the comment above (L287)

Reply:
@TomerShor I have a comment above